### DISCUSSION TITLE
Create a button to delete all CodeSpaces at once

### DISCUSSION THREAD

**Discussion by lab8-tomofi** [community/community#68368] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

I would like to create a button to delete all CodeSpaces at once. It is difficult to delete them one by one.

![image](https://github.com/community/community/assets/121648127/fddeaab6-45ce-453c-80f5-ac3a954013d8)

**Comment by vishalkrishna8** [comment, marked as answer]
Hello @lab8-tomofi ,
GitHub didn't have a built-in way to delete all CodeSpaces at once. You typically had to delete them one by one manually.
However, you can use the GitHub API and some scripting to automate this process.
let me know if this was helpful to you.
